docs/develop/java/testing-suite.mdx

```diff
@@ -301,7 +308,7 @@ Learn to set up the time-skipping test framework in the SDK of your choice.
 
 You can skip time automatically in the SDK of your choice.
 Start a test server process that skips time as needed.
-For example, in the time-skipping mode, Timers, which include sleeps and conditional timeouts, are fast-forwarded except when Activities are running.
+For example, in the time-skipping mode, Timers, which include sleeps and conditional timeouts, are fast-forwarded except when Activities or Nexus Operations are running.
```

Comment:
I would adjust a bit and add a caveat that nexus operation handlers timeout after 10 seconds.